frontend/src/bundles/talent-onboarding/components/bread-crumbs/styles.module.scss

```diff
@@ -0,0 +1,32 @@
+.breadCrumbs {
+    display: flex;
+    align-items: center;
+    height: 70px;
+    padding-left: 50px;
+    border-bottom: 1px solid var(--solitude);
+}
+
+.breadCrumbsList {
+    display: flex;
+    align-items: center;
+    padding-right: 5px;
+    list-style-type: none;
+}
+
+.breadCrumbLink {
+    font-weight: var(--font-weight-regular);
+    font-size: var(--font-size-body);
+}
+
+.breadCrumbLinkActive {
+    color: var(--st-tropaz);
+    font-weight: var(--font-weight-medium);
+}
+
+a[aria-current="page"] {
+    color: var(--text-primary);
+}
```

Comment:
this breaks sidebar icons styling on current page (white color). need to fix styles conflicts

Reply:
done